components/raftstore/src/store/snap.rs

```diff
@@ -1896,9 +1912,12 @@ impl SnapManagerBuilder {
         } else {
             u64::MAX
         };
+        let path = path.into();
+        let mut path_v2 = path.clone();
+        path_v2.push_str("_v2");
```

Comment:
Does it cause compatibility issues? E.g., upgrade from an old tikv, can it read or clean up old snapshots.

Reply:
this is the path for v2's snap mgr, v1's snap mgr will not be changed.